app/policies/group_type_policy.rb

```diff
@@ -0,0 +1,21 @@
+class GroupTypePolicy < ApplicationPolicy
```

Comment:
it looks like asking for some refactoring, how about class like AdminCrudPolicy < ApplicationPolicy put below stuff there and inherit from that class ?

Reply:
I wasn't sure how much these policies would change and differ in the future, e.g. will leads be able to edit users in their teams or edit actuals for users in their teams etc. But for now it is just admins so this is a good shout.